go.mod

```
@@ -6,11 +6,16 @@ require (
 	github.com/allegro/bigcache v1.2.1 // indirect
 	github.com/ethereum/go-ethereum v1.8.27
 	github.com/fjl/memsize v0.0.0-20190710130421-bcb5799ab5e5 // indirect
+	github.com/golang-migrate/migrate/v4 v4.6.1 // indirect
 	github.com/google/uuid v1.1.1
 	github.com/karalabe/hid v1.0.0 // indirect
 	github.com/pkg/errors v0.8.1
 	github.com/rjeczalik/notify v0.9.2 // indirect
-	github.com/status-im/status-go v0.29.0-beta.3
-	github.com/status-im/status-protocol-go v0.0.0-20190624072549-57675d19152c73af930b1e8f93d298322cbfe95c
+	github.com/status-im/migrate/v4 v4.3.1-status // indirect
+	github.com/status-im/status-go v0.30.1-beta.2.0.20190828210454-4761179cc067
```

This doesn't work, you have to specify a full commit hash:
```suggestion
	github.com/status-im/status-go v0.30.1-beta.2.0.20190828210454-4761179cc0673a0bb8fe0d33ca6a035ca62c28c7
```

---

I fixed it myself.